### PR TITLE
Issue 7297 - Amend static query code

### PR DESCRIPTION
**Pull request by nhc**
This is a pull request for this issue https://github.com/gatsbyjs/gatsby/issues/7297

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 4350d417d208d8da9a4906fb0a58af891ec123df

https://app.netlify.com/sites/gatsbyjs/deploys/5b72eb07c96592545214155e

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 4350d417d208d8da9a4906fb0a58af891ec123df

https://app.netlify.com/sites/using-postcss-sass/deploys/5b72eb08c96592545214156a

File: docs/docs/static-query.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ Video hosted on [egghead.io][egghead].
...
-import { StaticQuery, graphql } from "gatsby"
+import React from 'react';
+import { StaticQuery, graphql } from 'gatsby';
 
-const Header = () => (
+export default () => (
   <StaticQuery
     query={graphql`
-      query {
+      query HeadingQuery {
```
Comment (m-allanson) [inline docs/docs/static-query.md]: A question for @jlengstorf or @amberleyromo - technically we don't need to name queries any more. In this case I guess we should leave this in so the code matches the video?

Comment (jlengstorf) [reply docs/docs/static-query.md]: Yeah, I didn't know that when I made the video. We could potentially call it out, but it doesn't do any harm, right? I think we can leave this one for now.

**Pull request merged**